src/MeshIntegrals.jl

```diff
@@ -1,8 +1,8 @@
 module MeshIntegrals
-using CoordRefSystems
-using LinearAlgebra
-using Meshes
-using Unitful
+using CoordRefSystems: CoordRefSystems, CRS
+using LinearAlgebra: LinearAlgebra, I, norm
+using Meshes: Meshes, Line, Plane, Ray, area, degree, plane, segments, volume, ×, ⋅
```

Comment:
Where are `×` and `⋅` products defined on `Meshes.vec` vectors? I had assumed that Meshes was defining methods on `LinearAlgebra.:×` and `LinearAlgebra.:⋅` but I don't see any defined in the codebase. However, when I `using Meshes` in the REPL and check the namespace it does seem like they're exporting the symbols, though the docstrings are just the ones from LinearAlgebra.

Reply:
Yeah, I was also a bit confused. I think Meshes doesn't need to define methods for `⋅` and `×` because in LinearAlgebra (see e.g. https://github.com/JuliaLang/julia/blob/master/stdlib/LinearAlgebra/src/generic.jl#L922) they are already generic and [`Vec`](https://github.com/JuliaGeometry/Meshes.jl/blob/master/src/vectors.jl#L40) from Meshes.jl inherites `SVector`. So the methods from LinearAlgebra are applicable. This means Meshes.jl basically only reexports them from LinearAlgebra. I guess it doesn't matter whether we import them from LinearAlgebra or from Meshes. Maybe it is nicer to imprt them from LinearAlgebra because this is where they are defined? I let you decide.

Reply:
That was my thinking too, that they just inherit a method from `SVector`. I'm going to try it out on LinearAlgebra. If everything checks out, then I think we should go with that.